### PR TITLE
titles modified

### PR DESCRIPTION
**Pull request by jstnk9**
Other titles modified :)

### CODE REVIEW

**Review by nasbench — APPROVED** [review]

File: rules/windows/registry/registry_set/registry_set_abusing_windows_telemetry_for_persistence.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-title: Abusing Windows Telemetry For Persistence
+title: Abusing Windows Telemetry For Persistence - Registry Set
```
Comment (frack113) [inline rules/windows/registry/registry_set/registry_set_abusing_windows_telemetry_for_persistence.yml:1]: `title: Abusing Windows Telemetry For Persistence - Registry`
Like the other rules
need to fix the 2 last 
- rules/windows/registry/registry_set/registry_set_renamed_sysinternals_eula_accepted.yml:title: Usage of Renamed Sysinternals Tools - RegistrySet
- rules/windows/registry/registry_set/registry_set_uac_bypass_eventvwr.yml:title: UAC Bypass via Event Viewer - Registry Set

File: rules/windows/powershell/powershell_script/posh_ps_wmi_unquoted_service_search.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-title: WMIC Unquoted Services Path Lookup
+title: WMIC Unquoted Services Path Lookup - Powershell Script Block Text
```
Comment (frack113) [inline rules/windows/powershell/powershell_script/posh_ps_wmi_unquoted_service_search.yml:1]: `title: WMIC Unquoted Services Path Lookup - PowerShell`
Like the other rules

File: rules/windows/powershell/powershell_script/posh_ps_susp_local_group_reco.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-title: Suspicious Get Local Groups Information
+title: Suspicious Get Local Groups Information - Powershell Script Block Text
```
Comment (frack113) [inline rules/windows/powershell/powershell_script/posh_ps_susp_local_group_reco.yml:1]: `title: Suspicious Get Local Groups Information - PowerShell`
Like the other rules

File: rules/windows/powershell/powershell_script/posh_ps_etw_trace_evasion.yml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-title: Disable of ETW Trace
+title: Disable of ETW Trace - Powershell Script Block Text
```
Comment (frack113) [inline rules/windows/powershell/powershell_script/posh_ps_etw_trace_evasion.yml:1]: `title: Disable of ETW Trace - PowerShell`
Like the other rules

File: rules/windows/powershell/powershell_script/posh_ps_cl_invocation_lolscript.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-title: Execution via CL_Invocation.ps1
+title: Execution via CL_Invocation.ps1 - Powershell Script Block Text
```
Comment (frack113) [inline rules/windows/powershell/powershell_script/posh_ps_cl_invocation_lolscript.yml:1]: `title: Execution via CL_Invocation.ps1 - PowerShell`
Like the other rules

File: rules/windows/registry/registry_set/registry_set_abusing_windows_telemetry_for_persistence.yml (hunk 1 above)
Comment (jstnk9) [reply rules/windows/registry/registry_set/registry_set_abusing_windows_telemetry_for_persistence.yml]: Modified

File: rules/windows/powershell/powershell_script/posh_ps_wmi_unquoted_service_search.yml (hunk 2 above)
Comment (jstnk9) [reply rules/windows/powershell/powershell_script/posh_ps_wmi_unquoted_service_search.yml]: Modified

File: rules/windows/powershell/powershell_script/posh_ps_susp_local_group_reco.yml (hunk 3 above)
Comment (jstnk9) [reply rules/windows/powershell/powershell_script/posh_ps_susp_local_group_reco.yml]: Modified

File: rules/windows/powershell/powershell_script/posh_ps_etw_trace_evasion.yml (hunk 4 above)
Comment (jstnk9) [reply rules/windows/powershell/powershell_script/posh_ps_etw_trace_evasion.yml]: Modified

File: rules/windows/powershell/powershell_script/posh_ps_cl_invocation_lolscript.yml (hunk 5 above)
Comment (jstnk9) [reply rules/windows/powershell/powershell_script/posh_ps_cl_invocation_lolscript.yml]: Modified

**Pull request merged**